x/lightclient/types/state_test.go

```diff
@@ -0,0 +1,129 @@
+package types_test
+
+import (
+	"testing"
+	"time"
+
+	errorsmod "cosmossdk.io/errors"
+	ibcclienttypes "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
+	"github.com/dymensionxyz/dymension/v3/x/lightclient/types"
+	rollapptypes "github.com/dymensionxyz/dymension/v3/x/rollapp/types"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	timestamp     = time.Unix(1724392989, 0)
+	validIBCState = types.IBCState{
+		Root:               []byte("root"),
+		Timestamp:          timestamp,
+		NextValidatorsHash: []byte{156, 132, 96, 43, 190, 214, 140, 148, 216, 119, 98, 162, 97, 120, 115, 32, 39, 223, 114, 56, 224, 180, 80, 228, 190, 243, 9, 248, 190, 33, 188, 23},
+		ValidatorsHash:     []byte{156, 132, 96, 43, 190, 214, 140, 148, 216, 119, 98, 162, 97, 120, 115, 32, 39, 223, 114, 56, 224, 180, 80, 228, 190, 243, 9, 248, 190, 33, 188, 23},
+	}
+	validRollappState = types.RollappState{
+		BlockSequencer: []byte{10, 32, 86, 211, 180, 178, 104, 144, 159, 216, 7, 137, 173, 225, 55, 215, 228, 176, 29, 86, 98, 130, 25, 190, 214, 24, 198, 22, 111, 37, 100, 142, 154, 87},
+		BlockDescriptor: rollapptypes.BlockDescriptor{
+			StateRoot: []byte("root"),
+			Timestamp: timestamp,
+		},
+		NextBlockSequencer: []byte{10, 32, 86, 211, 180, 178, 104, 144, 159, 216, 7, 137, 173, 225, 55, 215, 228, 176, 29, 86, 98, 130, 25, 190, 214, 24, 198, 22, 111, 37, 100, 142, 154, 87},
+		NextBlockDescriptor: rollapptypes.BlockDescriptor{
+			StateRoot: []byte("root2"),
```

Comment:
can you document where these bytes come from?

Maybe it would be clearer to derive this stuff from some example private keys?